DataConnectors/Beyond Security beSECURE.json

```diff
@@ -0,0 +1,109 @@
+{
+  "id": "BeyondSecuritybeSECURE",
+  "title": "Beyond Security beSECURE",
+  "publisher": "Beyond Security",
+  "descriptionMarkdown": 
+  "The [Beyond Security beSECURE](https://beyondsecurity.com/) connector allows you to easily connect your Beyond Security beSECURE scan events, scan results and audit trail with Azure Sentinel, to view dashboards, create custom alerts, and improve investigation. This gives you more insight into your organization's network and improves your security operation capabilities.",
+  "graphQueries": [
+    {
+      "metricName": "Total data received",
+      "legend": "Beyond Security beSECURE",
+      "baseQuery": "beSECURE_ScanResults_CL"
+    }
+  ],
+  "sampleQueries": [
+    {
+      "description" : "CVSS Score Aggregated",
+      "query": "beSECURE_ScanResults_CL  | summarize count() by cvss_score_s"
+    },
+	{
+	  "description" : "Risk Aggregated",
+	  "query": "beSECURE_ScanResults_CL  | summarize count() by risk_name_s"
+	},
+	{
+	  "description": "Host Risk Aggregated",
+	  "query": "beSECURE_ScanResults_CL  | summarize count() by destination_host_s, risk_name_s"
+	},
+	{
+	  "description" : "Top 10 most vulnerable Hosts",
+	  "query": "beSECURE_ScanResults_CL  | \n where (risk_value_s == 8 or risk_value_s == 4) | \n summarize count() by destination_host_s, risk_value_s | \n order by risk_value_s, count_ | \n limit 10"
+	}
+  ],
+  "dataTypes": [

```

Comment:
If you have 3 data types - beSECURE_Audit_CL, beSECURE_ScanEvents_CL, beSECURE_ScanResults_CL you need to call out all 3. I guessed the data types based on the sample data files you have. You can represent multiple data types as follows:
   "dataTypes": [
        {
            "name": "beSECURE_Audit_CL",
            "lastDataReceivedQuery": "beSECURE_Audit_CL\n            | summarize Time = max(TimeGenerated)\n            | where isnotempty(Time)"
        },
        {
            "name": "beSECURE_ScanEvents_CL",
            "lastDataReceivedQuery": "beSECURE_ScanEvents_CL\n            | summarize Time = max(TimeGenerated)\n            | where isnotempty(Time)"
        },
        {
            "name": "beSECURE_ScanResults_CL",
            "lastDataReceivedQuery": "beSECURE_ScanResults_CL\n            | summarize Time = max(TimeGenerated)\n            | where isnotempty(Time)"
        },
    ],